examples/dt/compute/nfs/readme.md

```diff
@@ -0,0 +1,67 @@
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## Purpose
+
+This DT targets a normal topology for the compute with NFS backend.
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Ceph                                       | vm    | 3      |
+
+## Services, enabled features and configurations
+
+| Service         | configuration                | Lock-in coverage?  |
+| ----------------| ---------------------------- | ------------------ |
+| Neutron         | ML2/OVN, Geneve              | Must have          |
+| Compute         | set-nova-scheduler-filter    | Must have          |
+| Cinder          | NFS                          | Must have          |
```

Comment:
there is already a proposed DT with cinder and several other services over NFS; couldn't that one be used for this too? https://github.com/openstack-k8s-operators/architecture/pull/45/files#diff-d14bc086c95e6bb1914281cdaead3abb1033b8300bea0d603e04ff5be3392531

Reply:
Will discuss on what tests to include in the Dt's proposed.